sonar-project.properties

```diff
@@ -0,0 +1,6 @@
+sonar.organization=polymathnetwork-1
+sonar.projectKey=PolymathNetwork_polymesh-sdk
+sonar.sources=.
+sonar.javascript.environments=node
```

Comment:
Shouldn't all these be `sonar.typescript.blah`?

Reply:
It dependes.
For coverage:  `sonar.javascript.lcov.reportPaths | Comma-delimited list of paths to LCOV coverage report files. Paths may be absolute or relative to project root.`
For TSLint: `sonar.typescript.tslint.reportPaths`